index.html

```diff
@@ -0,0 +1,1209 @@
+                        <div datatype="rdf:HTML" property="schema:description">
+                          <p>Implementations may perform cleanup tasks such as deleting the ACL resource that is associated with a resource when the resource is deleted. As deleted resources may be replaced by new resources with the same URI, access permissions on the new resource may differ when the <cite><a href="#effective-acl-resource" rel="rdfs:seeAlso">Effective ACL Resource</a></cite> is determined.</p>
+                        </div>
+                      </div>
+
+                      <p>When an operation requests to read and write an ACL resource, the server MUST match an Authorization allowing the <code>acl:Control</code> access privilege on the resource.</p>
+
+                      <div class="note" id="http-method-access-mode-mapping" inlist="" rel="schema:hasPart" resource="#http-method-access-mode-mapping">
+                        <h5 property="schema:name"><span>Note</span>: HTTP Method and Access Mode Mapping</h5>
+                        <div datatype="rdf:HTML" property="schema:description">
+                          <p>When the target of the HTTP request is the ACL resource, the operation  can only be allowed with the <code>acl:Control</code> access mode.</p>
+
+                          <p>Having <code>acl:Control</code> does not imply that the agent has <code>acl:Read</code> or <code>acl:Write</code> access to the resource itself, just to its corresponding ACL resource. For example, an agent with control access may disable their own write access (to prevent accidental over-writing of a resource by an application), but be able to change their access levels at a later point (since they retain <code>acl:Control</code> access).</p>
+
+                          <p>The HTTP <code>GET</code> method request targeting a resource can only be allowed with the <code>acl:Read</code> access mode.</p>
+
+                          <p>The HTTP <code>POST</code> can be used to create a new resource in a container or add information to existing resources (but not remove resources or its contents) with either <code>acl:Append</code> or <code>acl:Write</code>.</p>
+
+                          <p>As the HTTP <code>PUT</code> method requests to create or replace the resource state, the <code>acl:Write</code> access mode would be required.</p>
+
+                          <p>As the processing of HTTP <code>PATCH</code> method requests depends on the request semantics and content, <code>acl:Append</code> can allow requests using SPARQL 1.1 Update’s [<cite><a href="#bib-sparql11-update">SPARQL11-UPDATE</a></cite>] <code>INSERT DATA</code> operation but not <code>DELETE DATA</code>, whereas <code>acl:Write</code> would allow both operations.</p>
+
+                          <p>As the HTTP <code>DELETE</code> method requests to remove a resource, the <code>acl:Write</code> access mode would be required.</p>
```

Comment:
For the rest of this note, I got a feeling this belongs more at the protocol level... 

It is good that WAC isn't tied to Solid, but OTOH, it might be that also implies that other protocols may have their own ideas on how to apply access modes, and so perhaps it shouldn't be there.

For Solid, OTOH, I think we need to have better language as to the implications of the various access modes on the protocol in the protocol spec. 

Anyway, this is more an itch in my aestethicle at this point, it isn't something that prevents merging.

Reply:
Glad you see it that way too. Just mentioned that at the end of the comment in https://github.com/solid/web-access-control-spec/pull/83#discussion_r656261006 . Related example on mapping: https://github.com/solid/web-access-control-spec/pull/83#discussion_r656068624

Once we are done here, my plan was to move the information in method-mode mapping / responses in https://github.com/solid/specification/issues/14#issuecomment-683480525 to the Protocol spec. It doesn't need to be tightly tied to WAC either. Assuming that "read", "write" (and maybe "append"), "control" are general class of operations that's useful for the Protocol.

Reply:
Excellent!